### ISSUE TITLE
Update libcrypto3/libssl3 to fix CVE-2023-2975

### ISSUE THREAD

**Issue by k725** [alpinelinux/docker-alpine#333]
It looks like #328.
I would appreciate it if you could update the base image.

```
┌────────────┬───────────────┬──────────┬───────────────────┬───────────────┬─────────────────────────────────────────────────────────────┐
│  Library   │ Vulnerability │ Severity │ Installed Version │ Fixed Version │                            Title                            │
├────────────┼───────────────┼──────────┼───────────────────┼───────────────┼─────────────────────────────────────────────────────────────┤
│ libcrypto3 │ CVE-2023-2975 │ LOW      │ 3.1.1-r1          │ 3.1.1-r2      │ Issue summary: The AES-SIV cipher implementation contains a │
│            │               │          │                   │               │ bug that c ......                                           │
│            │               │          │                   │               │ https://avd.aquasec.com/nvd/cve-2023-2975                   │
├────────────┤               │          │                   │               │                                                             │
│ libssl3    │               │          │                   │               │                                                             │
│            │               │          │                   │               │                                                             │
│            │               │          │                   │               │                                                             │
└────────────┴───────────────┴──────────┴───────────────────┴───────────────┴─────────────────────────────────────────────────────────────┘
```

**Comment by philipleetaylor** [comment]
We need libcrypto3 upgraded to 3.1.1-r3 now to fix yet another CVE:

- CVE-2023-2975 
- CVE-2023-3446

**Comment by philipleetaylor** [comment]
Both of these CVEs have been upgraded to medium risk.

- https://nvd.nist.gov/vuln/detail/CVE-2023-2975
- https://nvd.nist.gov/vuln/detail/CVE-2023-3446

**Comment by k725** [comment]
This issue has been fixed in 3.18.3 (latest).
